c++/src/RleEncoderV2.cc

```diff
@@ -577,7 +577,10 @@ void RleEncoderV2::writePatchedBasedValues(EncodingOption& option) {
     // find the number of bytes required for base and shift it by 5 bits
     // to accommodate patch width. The additional bit is used to store the sign
     // of the base value.
-    const uint32_t baseWidth = findClosestNumBits(option.min) + 1;
+    uint32_t baseWidth = findClosestNumBits(option.min) + 1;
```

Comment:
Seems like getClosestFixedBits method can indeed return up to 64 bits and adding thee sign bit here would exceed 8 bytes -- would it make sense to add the logic as part of the underlying method?

An alternative would be to skip the sign bit addition when the width is already 64 bits

Reply:
Change getClosestFixedBits() logic will affect another code paths.
Because the sign is only needed in PatchBase encoding.
So change the PatchBase encoding will be better.

Reply:
I will change it to skip the sign bit addition when the width is already 64 bits